packages/react-composites/src/composites/CallComposite/components/CallPane.tsx

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IStackStyles, IStackTokens, ITheme, Stack } from '@fluentui/react';
+import {
+  _DrawerMenu,
+  _DrawerMenuItemProps,
+  _useContainerHeight,
+  _useContainerWidth,
+  ParticipantMenuItemsCallback,
+  useTheme
+} from '@internal/react-components';
+import React, { useMemo, useRef, useState } from 'react';
+import { CallAdapter } from '../';
+import { CallAdapterProvider } from '../adapter/CallAdapterProvider';
+import { AvatarPersonaDataCallback } from '../../common/AvatarPersona';
+import {
+  paneBodyContainer,
+  scrollableContainer,
+  scrollableContainerContents
+} from '../../common/styles/ParticipantContainer.styles';
+import { SidePaneHeader } from '../../common/SidePaneHeader';
+import { ModalLocalAndRemotePIP, ModalLocalAndRemotePIPStyles } from '../../common/ModalLocalAndRemotePIP';
+import { PeoplePaneContent } from '../../common/PeoplePaneContent';
+import { drawerContainerStyles } from '../styles/CallComposite.styles';
+import { TabHeader } from '../../common/TabHeader';
+import { _ICoordinates } from '@internal/react-components';
+import { _pxToRem } from '@internal/acs-ui-common';
+import { useLocale } from '../../localization';
+
+/**
+ * Pane that is used to store participants for Call composite
+ * @private
+ */
+/** @beta */
+export const CallPane = (props: {
+  callAdapter: CallAdapter;
+  onClose: () => void;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  onPeopleButtonClicked?: () => void;
+  modalLayerHostId: string;
+  activePane: CallPaneOption;
+  mobileView?: boolean;
+  inviteLink?: string;
+  rtl?: boolean;
+}): JSX.Element => {
+  const [drawerMenuItems, setDrawerMenuItems] = useState<_DrawerMenuItemProps[]>([]);
+
+  const hidden = props.activePane === 'none';
+  const paneStyles = hidden ? hiddenStyles : props.mobileView ? availableSpaceStyles : sidePaneStyles;
+
+  const useStrings = () => {
```

Comment:
This is used to resolve issues with conditional compilation complaining that CallCompositeStrings do not have the necessary strings. Once out of beta, this will be changed to only use CallCompositeStrings.